app/packs/src/components/common/CommentModal.js

```diff
@@ -292,22 +290,27 @@ export default class CommentModal extends Component {
               }
             </div>
 
-            <FormControl
-              componentClass="textarea"
+            <Form.Control
+              as="textarea"
+              rows={5}
               autoFocus
-              style={{
-                height: '100px',
-                marginBottom: '20px',
-              }}
               value={commentBody}
-              ref={(input) => { this.nameInput = input; }}
-              inputRef={(m) => {
+              // ref={(input) => { this.nameInput = input; }}
```

Comment:
I'd be fine with removing this line. `this.nameInput` does not seem to be referenced anywhere.